Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -292,7 +292,10 @@ protected async Task OnTextKeyDownHandler( KeyboardEventArgs eventArgs )
             }
 
             if ( ActiveItemIndex.HasValue )
+            {
+                await Task.Yield();
```

Comment:
If you added this then I already know what the problem is... you were testing in WASM, right? 
I was testing in Blazor.Server, that's probably why I wasn't having the same bug.
So we should change this check to only target wasm. 
However Gotta experiment with the js part, see if we can avoid the Task.Yield, a negligible amount of delay on js or smthg else, see if it helps instead.

Reply:
Oh I see that you removed the Task.Yield() in the next commit, `ScrollIntoView` in favor of `ScrollTo`.
 What's the fundamental difference between these? We use this in other parts also.

Reply:
Nvm, I see... One you need to tell it the coordinates instead of the inner api calculating it for you. I'll test it out on the other places, see if it remains equivalent. Thanks.

Reply:
there's a similar issue with Focus() I found in code (https://github.com/dotnet/aspnetcore/issues/30070#issuecomment-823938686) I wonder what the root cause for this is.. I'm not sure why scrollTo works when scrollElementIntoView does not.

Reply:
Just on of those weird things I guess. Yea I tested it works fine. Thanks for the fix.